frontend/test/metabase/scenarios/dashboard/bookmarks.cy.spec.js

```diff
@@ -0,0 +1,33 @@
+import { restore, sidebar } from "__support__/e2e/cypress";
+
+describe("scenarios > dashboard > bookmarks", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should update the name and description", () => {
+    cy.visit("/dashboard/1");
+
+    cy.icon("ellipsis").click();
+
+    cy.findByText("Bookmark").click();
+
+    cy.visit("/collection/root");
+
+    sidebar().within(() => {
+      // Find the bookmark and click on it to visit dashboard page again
+      cy.findByText("Orders in a dashboard").click();
+    });
+
+    cy.icon("ellipsis").click();
+
+    cy.findByText("Remove bookmark").click();
+
+    cy.visit("/collection/root");
+
+    sidebar().within(() => {
+      cy.findByText("Orders in a dashboard").should("not.exist");
```

Comment:
This can easily result in a false positive.

Reply:
@nemanjaglumac Checking this, I'm led to believe that in this case no.

I think not because it existed before (see line 20).

Reply:
Really depends on the machine that it's being run on. If requests take forever and the contents still doesn't load (let's say you have a loading spinner), Cypress would've asserted that this string doesn't exist.

Reply:
Yes, you're right. Updated.